docs/UserGuide.md

```diff
@@ -108,74 +108,56 @@ The list of companies should be listed in the following format below:
 ```
 ### Deleting a company : `delete` ###
 
-Deletes a company from the address book.
-
 **Format:`delete INDEX`**
 
-* Deletes the contact at the specific INDEX
-* The index refers to the index number shown in the displayed contact list
+You add an entry by mistake or you no longer want to track a particular company, 
+you can delete it using the delete command. Which deletes the company at the specified INDEX.
+
+* The index refers to the index number shown in the displayed company list
 * The index must be a positive integer
+* The index must not be out of bounds.
+* You can only delete one company at a time.
 
 **Examples:**
 * `list` followed by `delete 1` deletes the first contact in the full list
 * `find` TikTok followed by `delete 1` deletes the first contact in the results of find TikTok
```

Comment:
Will results of find TikTok be confusing? since find command is not yet introduced